transfers/alchemy_getAssetTransfers.yaml

```diff
@@ -35,7 +40,7 @@ paths:
         content:
           application/json:
             schema:
-              $ref: evm_body.yaml#/alchemy_getAssetTransfers # path for using github actions to update, to update from CLI using rdme change to ../evm_body.yaml#/alchemy_getAssetTransfers
+              $ref: ../evm_body.yaml#/alchemy_getAssetTransfers # path for using github actions to update, to update from CLI using rdme change to ../evm_body.yaml#/alchemy_getAssetTransfers
```

Comment:
we need to keep this as is so that we can use for github actions. you can change it to ../ for testing but it should be the way it was before when submitting to master branch